images/ko/configs/latest.apko.yaml

```diff
@@ -1,4 +1,8 @@
 contents:
+  repositories:
```

Comment:
I'm confused about the changes to this file, I would have expected this to only have red lines?

Reply:
Ah, Josh dropped these when he ported `ko` to TF and we haven't been consistent about dropping these everywhere so I brought them back for now.  I'm pretty sure this would pass without this because of the variables, but that was some of my reasoning fwiw.